azkaban-common/src/main/java/azkaban/user/Permission.java

```diff
@@ -191,6 +191,7 @@ public int toFlags() {
     SCHEDULE(0x0000008),
     METRICS(0x0000010),
     CREATEPROJECTS(0x40000000), // Only used for roles
+    UPLOADPROJECTS(0x0008000),
```

Comment:
Does it make sense that admins can't upload when lockdown.upload.projects is set to true? Why?

---

How are users granted UPLOADPROJECTS permissions?